clmm/dataops.py

```diff
@@ -0,0 +1,275 @@
+"""Functions to compute polar/azimuthal averages in radial bins"""
+# try: # 7481794
+#     import pyccl as ccl
+# except ImportError:
+#     pass
+import math
+import warnings
+import numpy as np
+from .gcdata import GCData
+from .utils import compute_radial_averages, make_bins, convert_units
+from .modeling import get_critical_surface_density
+def compute_tangential_and_cross_components(
```

Comment:
I noticed there are no blank lines above any function definitions in `dataops.py` or in the methods of `galaxycluster.py`. Is it due to some autoformatting at your end? I find it  makes the code hard to read. Also I think PEP8 recommands two blank kines between function definitions.

Reply:
no problem, I will change it